### DISCUSSION TITLE
Welcome to ess-cardapio Discussions!

### DISCUSSION THREAD

**Discussion by crvzybird** [crvzybird/ess-cardapio#9] [Announcements]
## 👋 Bem vindo(a)!
  Estamos usando **_Discussions_**  como um espaço para debater implementações, realizar **_Brainstorms_** de **_Features_** e qualquer tópico relacionado ao desenvolvimento desse projeto com o intuito de podermos nos manter conectados mesmo não sendo possível estar em reuniões virtuais/presenciais. É esperado que você :
  * Faça perguntas, não fique em dúvida sobre o que quer que seja do projeto .
  * Compartilhe ideias e faça sugestões.
  * Interaja com os tópicos relacionados à sua parte do projeto e também em discussões gerais.

  Para começar, comente aqui embaixo qual **_Feature_**  você gostaria de estar contribuindo e participando do seu próprio time de desenvolvimento.Elas são:

-   :pencil2:  Cadastro e Manutenção
-   :bookmark: Criação de Categorias
-   :link: Compartilhamento de Link

  :bulb: Para outras diretrizes e boas práticas, consultar a documentação do Github sobre [Discussions](https://docs.github.com/en/discussions/guides/best-practices-for-community-conversations-on-github)

**Comment by costa402** [comment]
✏️ Cadastro e Manutenção

**Comment by crvzybird** [comment]
:pencil2: Cadastro e Manutenção

**Comment by mateusmeloxyz** [comment]
Compartilhar cardápio

**Comment by leehmaciel** [comment]
✏️ Cadastro e Manutenção

**Comment by mateussgn** [comment]
Criação de categorias de itens
